smoketest/scripts/cli/test_protocols_ospf.py

```diff
@@ -382,10 +382,10 @@ def test_ospf_13_export_list(self):
 
 
     def test_ospf_14_segment_routing_configuration(self):
-        global_block_low = "100"
-        global_block_high = "199"
-        local_block_low = "200"
-        local_block_high = "299"
+        global_block_low = "300"
```

Comment:
This file is for OSPF but the PR and commit message is for ISIS.

Reply:
Yes, so, if you'd like I can make another one but I figure since it's a smoketest and since this issue could very likely affect both ISIS and OSPF I went and I did to both. If you'd like I can do them separately. Or I can rename the task, and the PR here as well.

Reply:
No need to a dedicated PR. Just move it to a dedicated commit.

Reply:
I am....not quite sure what you mean my good sir. Should I revert this and just commit it afterwards? Or should I do multiple commits on this PR? 

I've always been under the impression I am supposed to squash all commits to VyOS....

Reply:
Well a Github PullRequest can consist out of multiple commits. As in this case you should get at least two commits, one for ISIS and the other one for the OSPF smoketest adjustment. GitHub PullRequests are Githubs way on how to integrate 3rd party changes into another repository.

In general it is always good to have git commits focus on a single topic
- work in isis stuff
- change the ospf smoketest

You can split your commit into multiple commits using `git rebase -i HEAD~1` or `git reset --mixed HEAD~1` and restructure your index.

> I've always been under the impression I am supposed to squash all commits to VyOS....

Actually this defeats the "one commit per topic" approach. Some projects like to squash a PR into one single commit. The big downside is that it makes project maintenance harder as you can no longer use `git cherry-pick` to backport certain features. Also you loose full track of the projects history/how that code was developed when a repository is e.g. moved away from GitHub and you no londer have the web based PRs available. PM me in Slack if you have additional questions.